### PR TITLE
Revert removal of addr and peeraddr delegates

### PR DESCRIPTION
**Pull request by d-snp**
Fixes the issue reported in #166

### CODE REVIEW

**Comment by digitalextremist** [comment]
@d-snp have you verified this works as expected? I seem to remember that on some versions of the socket the `peeraddr` method does not return the information anticipated. If you can confirm this does work, I'll merge it in.

**Comment by d-snp** [comment]
I added some silly specs to confirm the existance of the delegators and that they respond the peer addr tuples. It seems to be working but I haven't tested it outside of the specs. If the socket wasn't a true TCPSocket then I think the peeraddr/addr methods would be missing, that's what you were getting at right? Or is there something I should test out in a real world scenario?

**Comment by kenichi** [comment]
great work on the wsdriver integration - i've updated angelo to work with this PR :+1:

when this merges, i'll merge https://github.com/kenichi/angelo/pull/50 and update Gemfile

**Comment by digitalextremist** [comment]
Looks good @d-snp! Merging @kenichi, so you ought to be good to go.

File: lib/reel/websocket.rb [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
 module Reel
   class WebSocket
-    include Celluloid::Logger
+    extend Forwardable
     include ConnectionMixin
     include RequestMixin
 
-    attr_accessor :socket
-    
+    attr_reader :socket
```
Comment (digitalextremist) [inline lib/reel/websocket.rb]: One afterthought @d-snp ... are there no cases where we'll need this to be `attr_writer` or `attr_accessor` as it was?

**Pull request merged**